.github/workflows/create-draft-release.yaml

```diff
@@ -24,64 +24,68 @@ jobs:
     - name: Create or update draft release tags
       run: |
         git fetch --tags
-        for aws_release_dir in capa/*; do
-          if [[ -f $aws_release_dir ]]; then
-            echo "Skipping file $aws_release_dir"
-            continue
-          fi
-          aws_release_version="$(basename $aws_release_dir)"
-          echo "Found AWS release $aws_release_version"
-          draft_tag_name="draft/aws/$aws_release_version"
-          if [ $(git tag -l "$draft_tag_name") ]; then
-            echo "Found existing tag $draft_tag_name, will force-overwrite it"
-            # Delete origin tag
-            git push --delete origin "$draft_tag_name"
-            # Create new tag locally
-            git tag -fa "$draft_tag_name" -m "AWS draft release $aws_release_version"
-            git push -f origin "$draft_tag_name"
-          else
-            echo "Tag $draft_tag_name not found"
-            
-            published_tag_name="aws/$aws_release_version"
-            published_release=$(gh release --repo giantswarm/releases view "$published_tag_name" --json "id,name,tagName,assets,isDraft,isPrerelease,url" 2>&1 || true)
-            if [ "$published_release" = "release not found" ]; then
-              echo "Didn't find a release for neither published tag $published_tag_name nor for tag $draft_tag_name. Will create a new draft tag."
-              git tag -a "$draft_tag_name" -m "AWS (CAPA) draft release $aws_release_version"
+          for provider_dir in capa/* azure/*; do
```

Comment:
you can drop this change in `.github/workflows/create-draft-release.yaml`, as I am removing this in another PR.

Reply:
done, not touching it 👍🏻